spot-ml/ml_ops.sh

```diff
@@ -97,7 +105,7 @@ time spark-submit --class "org.apache.spot.SuspiciousConnects" \
   --dupfactor ${DUPFACTOR} \
   --feedback ${FEEDBACK_PATH} \
   --ldatopiccount ${TOPIC_COUNT} \
-  --userdomain ${USER_DOMAIN}\
+  $USER_DOMAIN_PARSER_CMD \
```

Comment:
Probably this is not the first time we set an optional parameter. This approach is nice, I'd just separate fixed parameter from optional for cosmetics reasons, that means to leave all the "--" parameters first and at the end any optional parameter.